### ISSUE TITLE
[macOS] vscode-server failing to start

### ISSUE THREAD

**Issue by superman-lopez** [microsoft/vscode-remote-release#2905]
After one of the upgrades in the last month (not sure which), the VS Code Remote no longer works for me.  My host is macOS Catalina (zsh is the default shell, but bash is also installed), my clients are macOS Catalina (zsh) and Mojave (bash).  When connecting to the host, the connection is successful and the VS Code Server is downloaded and installed however it fails to start.  My version of `Remote - SSH` extension is `0.51.0`, and I also tried the nightly `2020.4.9540` with the same result.

Following other reported issues and articles, I tried the following:
Set the following config:
```js
"remote.SSH.showLoginTerminal": true,
"remote.SSH.useLocalServer": false
```
and run the following commands:
```
Kill VS Code Server on Host
```
```
Uninstall VS Code Server from Host
```
I also manually deleted the `~/.vscode-server` and `~/.vscode-remote` folders.

I have attached the Output on the failing connection. 
[vscode-remote.log](https://github.com/microsoft/vscode-remote-release/files/4562492/vscode-remote.log)

What could I try to resolve this?

**Comment by superman-lopez** [comment]
When I run the `server.sh` in a local shell on my host, the server application seems to start fine.  I get greeted with the following:
```
IP Address: 10.0.1.30
Extension host agent listening on 8000


[20:08:47] Extension host agent started.
```

**Comment by superman-lopez** [comment]
When I first initiated the connection, I did this from my remote machine.  I suspect the connection was blocked by LuLu but because it was a remote connection I was not prompted (or somehow missed the prompt).  After uninstalling once more, and this time connecting from the host to `localhost` LuLu did prompt me.  The connection failed once more, but now I was able to accept the outgoing traffic in LuLu and consequent connections no longer failed.

So to use with LuLu: 1. Start with new installation of vscode-server. 2. While logged in on the host, open VS Code and connect to `localhost` to force LuLu to prompt about outgoing traffic. 3. Connect from intended client to host.
